admin/services/data-access/group.data.service.js

```diff
@@ -20,6 +25,15 @@ groupDataService.getGroups = async function (query) {
     .exec()
 }
 
+/**
+ * Get groups filtered by query.
+ * @returns {Promise<*>}
+ */
+groupDataService.sqlGetGroups = async () => {
+  const sql = `SELECT g.* FROM ${sqlService.adminSchema}.[group] WHERE isDeleted=0 SORT BY name ASC`
```

Comment:
you are aliasing the table as `g` but not creating the alias after the table name.

probably best to just remove the `g.` as you don't necessarily need it here